### ISSUE TITLE
console command $ ./X user:create throw errors.

### ISSUE THREAD

**Issue by soulge** [Brightcookie/lxHive#2]
# After enter the email address and password, error raised as below:
```php
PHP Warning:  Invalid argument supplied for foreach() in /home/vagrant/Code/lxHive/src/xAPI/Console/UserCreateCommand.php on line 58
PHP Stack trace:
PHP   1. {main}() /home/vagrant/Code/lxHive/X:0
PHP   2. Symfony\Component\Console\Application->run() /home/vagrant/Code/lxHive/X:50
PHP   3. Symfony\Component\Console\Application->doRun() /home/vagrant/Code/lxHive/vendor/symfony/console/Symfony/Component/Console/Application.php:126
PHP   4. Symfony\Component\Console\Application->doRunCommand() /home/vagrant/Code/lxHive/vendor/symfony/console/Symfony/Component/Console/Application.php:195
PHP   5. Symfony\Component\Console\Command\Command->run() /home/vagrant/Code/lxHive/vendor/symfony/console/Symfony/Component/Console/Application.php:874
PHP   6. API\Console\UserCreateCommand->execute() /home/vagrant/Code/lxHive/vendor/symfony/console/Symfony/Component/Console/Command/Command.php:257

Warning: Invalid argument supplied for foreach() in /home/vagrant/Code/lxHive/src/xAPI/Console/UserCreateCommand.php on line 58
```
API\Service\User  -- line 166

```php
public function fetchAvailablePermissions()
{
   $collection  = $this->getDocumentManager()->getCollection('authScopes');
   $cursor      = $collection->find();

    $this->cursor;

    return $this;
 }
```
change to

```php
public function fetchAvailablePermissions()
{
   $collection  = $this->getDocumentManager()->getCollection('authScopes');
   $cursor      = $collection->find();

    $this->cursor = $cursor;

    return $this;
 }
```

**Comment by RoboSparrow** [comment]
Hi soulge,

Thanks for notifying and providing the solution! We'll fix this as soon as possible.

**Comment by sraka1** [comment]
Thanks for the feedback, fixed in 443afb3aca803ddfeb6c7ccc2d3621aeb3856016.
